app/controllers/checkout_controller.rb

```diff
@@ -0,0 +1,42 @@
+class CheckoutController < ApplicationController 
+  def index
+    @cart= current_user.cart
```

Comment:
move to before action
```suggestion
    @cart = current_user.cart
```

Reply:
@order = current_user.orders.pending.last